### ISSUE TITLE
Prompts increase memory usage

### ISSUE THREAD

**Issue by BrechtCorbeel** [black-forest-labs/flux#127]
Anyone else notice this, very long prompts always destroy my VRAM.

**Comment by jeroenrijckaert** [comment]
No, I don't see any difference with Flux using Comfyui running --normalvram with about 12GB vram (I use the GGUF version of flux).

**Comment by BrechtCorbeel** [comment]
> No, I don't see any difference with Flux using Comfyui running --normalvram with about 12GB vram (I use the GGUF version of flux).

Other people noticed too I'm talking more then a page long prompts, it seems to have something to do with the fp8 clip model, when I switched to fp16, the prompt issue disappeared, but I stopped using workflows with that in general, because basically nothing worked with those.
